src/Core/Domain/Employee/Exception/CannotDeleteWarehouseManagerException.php

```diff
@@ -28,6 +28,8 @@
 
 /**
```

Comment:
Could you deprecate this method using recommended method : https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations ?